src/components/station/StationHistory.vue

```diff
@@ -234,7 +234,7 @@ export default defineComponent({
       const nextDate = new Date(d.toISOString());
       this.iterDate(nextDate);
       try {
-        const response = await fetchWithToken(`${window.VUE_APP_OAPI}/collections/${station.properties.topic}/items?f=json&datetime=${nextDate.toISOString()}/..&sortby=+resultTime&index=${index}&limit=1&wigos_station_identifier=${station.id}`);
+        const response = await fetchWithToken(`${window.VUE_APP_OAPI}/collections/${station.properties.topic}/items?f=json&datetime=${nextDate.toISOString()}/..&sortby=resultTime&index=${index}&limit=1&wigos_station_identifier=${station.id}`);
```

Comment:
Got rid of the `+` on the sort here since it was causing the fetch to fail. 

Not super clear if pygeoapi supports that. If I remove `+` it works fine. @webb-ben please let me know if I am missing something here.

Reply:
if no +/- indicated, + is assumed